src/rvsdg/egglog_optimizer/mod.rs

```diff
@@ -35,17 +36,18 @@ pub fn rvsdg_egglog_schedule() -> String {
     "(run-schedule
         ; It is sound to not saturate fast-analyses/subst, but we do because
         ; they won't blow up and will help other rules go through.
-        (repeat 5
-            (saturate fast-analyses)
-            (run)
-            (saturate subst))
+        (repeat 2
+            (run loop-strength-reduction)
```

Comment:
I don't think we need a loop-strength-reduction ruleset. Just add to the normal ruleset (don't include a ruleset on your rules)

Reply:
Removed